tsconfig.json

```diff
@@ -1,17 +1,11 @@
 {
```

Comment:
For reference, i’m switching to this btw: https://github.com/syntax-tree/hast-util-select/blob/main/tsconfig.json.

For `target`, would this normally result in a semver breaking change? Or does that not really matter because we require modern `vscode`?

Reply:
Updating `target` would normally be a part of dropping support for a specific Node.js version. It means the TypeScript code would be compiled to more modern JavaScript. I don’t even know if and how it affects JavaScript with types in jsdoc.

In this case it doesn’t matter anyway. The code is built using `esbuild`, explicitly targeted at `node16`. The same goes for other output options in https://github.com/syntax-tree/hast-util-select/blob/main/tsconfig.json

https://github.com/remarkjs/vscode-remark/blob/f9e6a496d0c37d72d6aa03ebb94a0c6fc4be1be2/package.json#L51